package.json

```diff
@@ -59,7 +62,7 @@
     "recursive-readdir": "2.1.0",
     "strip-ansi": "3.0.1",
     "style-loader": "0.13.1",
-    "truffle": "^5.0.14",
+    "truffle": "^5.1.53",
```

Comment:
What are we using Truffle for?

Reply:
I thought it was needed for the console but actually that only uses truffle contracts. I've removed this :)